app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] please, kindly ensure to enable `:Confirmable` in the user model. This will help to verify if an account is already confirmed to sign in, and to send emails with confirmation instructions.
- [ ] please kindly remove the comments to keep the file clean

Reply:
@tufoinnkuo10 Thanks for the professional review.

Reply:
- Excellent Implementation team 🥇 👍🏽 
<img width="1440" alt="Screenshot 2023-06-30 at 15 12 13" src="https://github.com/Besufekad-HAZ/Blog-App/assets/58692590/bb4d26bf-9ff8-4fcf-a4f1-8a59e247a43b">